shipyardbuild/circleci-orb
# chore: update python

To test:
- Use the circleCI SY orb `shipyard/shipyard@dev:alpha` on a pipeline
- Ensure tests pass

Additionally, connect to circle job via ssh and verify python defaults to python3. `python --version`
